src/Client.php

```diff
@@ -294,12 +311,30 @@ public function login()
             throw new CriticalException('Incorrect login to API, response does not have an access token!');
         }
 
-        $this->token = $data->accessToken;
+        $this->accessToken = $data->accessToken;
+        $this->refreshToken = $data->refreshToken;
         $this->loginFlag = false;
 
         return $this;
     }
 
+    public function refreshLogin()
+    {
+        try {
+            $response = $this->call('POST', $this->getEndpoint('login/refresh'), array(
+                    'refresh_token' => $this->refreshToken
+                )
+            );
+        } catch (\Exception $e) {
+            return;
+        }
+
```

Comment:
I have seen that in the login() call there is also a check if 'access_token' is not set in the response. Should this be also added for /login/refresh call?

Reply:
I think we can safely ignore that for the moment as the described case actually never occurred.